erpnext/controllers/accounts_controller.py

```diff
@@ -416,23 +416,22 @@ def company_abbr(self):
 		return self._abbr
 
 	def validate_party(self):
-		frozen_accounts_modifier = frappe.db.get_value( 'Accounts Settings', None,'frozen_accounts_modifier')
-		if frozen_accounts_modifier in frappe.get_roles():
-			return
-
 		party_type, party = self.get_party()
-
-		if party_type:
-			if frappe.db.get_value(party_type, party, "is_frozen"):
-				frappe.throw("{0} {1} is frozen".format(party_type, party), CustomerFrozen)
+		validate_party_frozen_disabled(party, party_type)
 
 	def get_party(self):
 		party_type = None
-		if self.meta.get_field("customer"):
-			party_type = 'Customer'
+		if self.doctype in ("Opportunity", "Quotation", "Sales Order", "Delivery Note", "Sales Invoice"):
+				party_type = 'Customer'
```

Comment:
@anandpdoshi That does not seem to be extra indentation. Since I checked in the local branch and it appears to be indented with 4 spaces.